src/Microsoft.TemplateEngine.Edge/Settings/SettingsLoader.cs

```diff
@@ -245,28 +221,8 @@ public void AddProbingPath(string probeIn)
             {
                 throw new ObjectDisposedException(nameof(SettingsLoader));
             }
-
-            const int maxAttempts = 10;
-            int attemptCount = 0;
-            bool successfulWrite = false;
-
-            while (!successfulWrite && attemptCount++ < maxAttempts)
-            {
-                if (!UserSettings.ProbingPaths.Add(probeIn))
-                {
-                    return;
-                }
-
-                try
-                {
-                    Save();
-                    successfulWrite = true;
-                }
-                catch
-                {
-                    Task.Delay(10).Wait();
-                }
-            }
+            EnsureLoaded();
```

Comment:
the method should be marked obsolete

Reply:
Do we need to mark this on implementation as well? I would assume interface is sufficient... Also this class is being deleted soon...